app/src/main/java/com/theuhooi/uhooipicbook/modules/monsterlist/MonsterListFragment.kt

```diff
@@ -24,12 +28,12 @@ class MonsterListFragment : Fragment() {
     // endregion
 
     // region View Life-Cycle Methods
```

Comment:
個人的な好みですが、 `region` コメントの下には空白行を入れたいです。

```suggestion
    // region View Life-Cycle Methods
    
```